gematria/basic_block/basic_block_protos.h

```diff
@@ -41,6 +41,12 @@ InstructionOperand InstructionOperandFromProto(
 CanonicalizedOperandProto ProtoFromInstructionOperand(
     const InstructionOperand& operand);
 
+// Creates a annotation data structure from a proto.
```

Comment:
Grammar nit: "an annotation"

Reply:
Ah, should've been more careful with the find-and-replacing of the old `RuntimeAnnotation` code :)